File: training/feuji/tasks/atm/atm.java
package tasks;

import java.util.*;
import java.util.Scanner;

public class atm {
	public static Scanner sc = new Scanner(System.in);
	public static int choice;
	public static boolean flag = false;
	static String i;
	static String j;
	static int row;
	static int column;
	static String[][] storedData = { { "21911026", "sam" }, { "21911018", "lav" }, { "21911012", "str" } };

	public static void loginAccDetails() {
		System.out.println("");
		System.out.println("*****WELCOME TO FEUJI ATM SERVICES*****");

		System.out.println("----------please login to your account-------");

		System.out.println("Please Enter Your Account Number :");
		i = sc.next();
		
		System.out.println("Please Enter Your Account password :");
		j = sc.next();
		validation(i, j);
	}
      
	public static void validation( String i, String j) {

		for (int row = 0; row < storedData.length; row++) {
			for (int column = 0; column < storedData[row].length; column++) {
				if (storedData[row][0].equals(i) && storedData[row][1].equals(j)) 
				{
					flag = true;
					System.out.println("successfully you have been logged in to your account");
				}
			}
			
		}
		do {

			if (flag) {
				System.out.println(" ");

				System.out.print("Welcome To Your Account:" + i);
				System.out.println(" ");
				System.out.println("Type 1 for Balance Checking");
				System.out.println("Type 2 for Deposit");
				System.out.println("Type 3 for Withdrawl");
				System.out.println("Type 4 for Logout");
				System.out.println("Type 5 for Exit");
				System.out.println("  ");
				System.out.println("Enter Your Choice : ");
			} else {
				System.out.print("Invalid account number or password");

				loginAccDetails();

			}

			choice = sc.nextInt();
			switch (choice) {
			case 1:
				CheckBalance();
				break;
			case 2:
				Deposit();
				break;
			case 3:
				Withdrawl();
				break;
			case 4:
				Logout();
				break;
			case 5:
				Exit();
			}
		} while (choice < 4);

	}

	public static void main(String[] args) {

		loginAccDetails();
	}

	public static void CheckBalance() {
		System.out.println("0.00");
	}

	public static void Deposit() {
		System.out.println("Your Deposits are Coming Soon");

	}

	public static void Withdrawl() {
		System.out.println("WITHDRAWLS ARE NOT YET STARTED");

	}

	public static void Logout() {
		System.out.println("Logout Succesfull");
		flag = false;
		loginAccDetails();
	}

	public static void Exit() {
		System.out.println("YOU HAVE BEEN EXITED SUCCESSFULLY");

	}
}

File: training/feuji/tasks/atm/atm_alert/atm.js
 var choice;
 var accNo,pwd;
 var flag=false;
 var deposit;
 var withdrawl;
 var accountNumbers=["21911018","21911026","21911043"];
 var passwords=["lav","sam","samanya"];
 var balance=["10000","20000","30000"];
 var choiceArray=[`
                 "Type 1 for Balance Checking"
                 " Type 2 for Deposit "
                 "Type 3 for Withdrawl"
                 "Type 4 for your Transaction history"
                 " Type 5 for Logout "
                 "Type 6 for Exit "`];
 var indexValue;
 var transactionHistory=[];
	var TransactionType;		   	
		var Amount;	 
			   
 
  
  
 
 
  function loginAccDetails(row)
  {
		        alert('*****WELCOME TO FEUJI ATM SERVICES*****');
			alert('----------please login to your account-------'); 
			 accNo =prompt("Please Enter Your Account Number :");
			 for (var row = 0; row<accountNumbers.length ; row++)
			 {
			   
			    
			      if(accNo==accountNumbers[row])
			         {
			               indexValue=row;
			               alert(indexValue);
			            pwd=prompt("Please Enter Your Account password :");
			            }
			            }
			            if(pwd==passwords[indexValue])
			            {
			               alert("successfully you have been logged in to your account");	                   
			               }
			                else
			               {
			               alert(" please enter valid details");
			                   return loginAccDetails();
			                   selectOptions();
			                   }
			
                                  
                         
                         
                         return indexValue;
        }
        // calling login process
	
var index=loginAccDetails();
        selectOptions();
                    
         function selectOptions()
                         
			{
                       alert("Welcome To Your Account:" + accNo);
		       alert(choiceArray+"\n");
	                var choice=prompt(`Enter Your Choice : ${choiceArray}`);
	                switch (choice)
	                   {
			      case "1":
				    CheckBalance();
				    selectOptions();
				break;
			      case "2":
				    Deposit();
				    selectOptions();
				break;
			      case "3":
				    Withdrawls();
				    selectOptions();
				break;
				case "4":
				      transHistory();
				      selectOptions();
			     case "5":
				    logout();
				    loginAccDetails();
				    selectOptions();
				break;
				
			     case "6":
			
				    Exit();
				    break;
			  }
			  return choice;
		    } 
		    
		    
              
	
		
	
	
 // checking your account balance
       function CheckBalance()
          {
            
            alert("your current balance is  "+balance[index]);
            
            }
         
  // to deposit 
        function Deposit() 
            {
                deposit=parseInt(prompt("please enter your deposit amount"));          	
            var transDeposit={TransactionType:" Depositing Amount",Amount:+deposit};
            transactionHistory.push(transDeposit);
          	balance[index]=parseInt(balance[index])+deposit;
          	alert(`your deposit has been successful.
          	your current balance is ${balance[index]} `);
          	
           }
      
  // to withdrawl
          function Withdrawls() 
            {
                  alert("your current  balance is"+balance[index] +"\n maximum amount to withdrawl is"+balance[index] );
                  withdrawl=prompt("please enter your withdrawl amount");
                     if(withdrawl<balance[index])
                         {
                         var transWithdrawl={TransactionType:" Withdrawl Amount",Amount:+withdrawl };
                         
                         
                         transactionHistory.push(transWithdrawl);
                         
                           var total=balance[index]-withdrawl;
                           alert("your remaining balance :"+total);
                           balance[index]=total;
                           
                        }
                    else
                        {
                          prompt("please withdraw according to your balance");
                        }
                     
             }  
               
               
   //function to get your transaction history
               function transHistory()
               {
                     var trans='';
                  for(var i=0;i<transactionHistory.length;i++)
                  {
                      trans +=`<tr>                     
                                  <td>${i+1}</td>
                                   <td> ${transactionHistory[i].TransactionType}</td> 
                                    <td>${ transactionHistory[i].Amount}</td>
                                </tr>`
                   }   
                         
                     
                       
                  }
                  alert(trans);
                 
               
   //function for logout from your account
            function logout() 
             {
             
                alert("Logout Succesfull");
                
             }
      
     
  // funtion to exit from the process
          function Exit() 
              {
       	         
		 alert("YOU HAVE BEEN EXITED SUCCESSFULLY");
              }
        
